hfendpoints-openai/src/lib.rs

```diff
@@ -33,8 +31,9 @@ const STATUS_DESC: &str = "Healthiness and monitoring of the endpoint";
         (status = OK, description = "Success", body = str, content_type = "application/json")
     )
 )]
-async fn health() -> Json<&'static str> {
-    Json::from("OK")
+#[instrument]
+async fn health() -> StatusCode {
+    StatusCode::OK
```

Comment:
Don't we end up in a situation where the underlying engine is booting but you advertise as healthy to the proxy so it sends requests?

Reply:
Yep that's definitely something I need to work on